### ISSUE TITLE
[Enhancement]  Search function - Jump to chat from result

### ISSUE THREAD

**Issue by Echolon** [iNPUTmice/Conversations#3101]
Hi,

would be great to for example hold on a result from in the search function and allow the user to directly jump to the actual context/chat.

Thus the user has access to its context, too. Often enough, you are looking for a whole conversation but only remember a single word.

Kind regards

**Comment by brighterness** [comment]
It would be great if bubbles had an analog of HREF Anchors.  Then quotes could link to content without quoting.

**Comment by laclaro** [comment]
Please realize this feature. Otherwise you can search specific messages, but often enough you know a keyword, but would like to know the context also, since your question is not in the exact message with the keyword you remember!

Best,

Henning

**Comment by anjandev** [comment]
Plaintext export existed in conversations < 2.4. It seems like implementing that again and allowing users to grep (outside of conversations) it will be a good solution for implementing this feature without requiring a refactor.

**Comment by eveofdestruction** [comment]
to avoid memory slurp consider this function only available via plugin - explicit opt-in for higher resource consumption

**Comment by weiss** [comment]
I would still very much appreciate being able to somehow see the context of search results.  But yes I guess the issue can be closed if nobody is motivated to implement this (maybe because it's too hard/impossible with the current design).

**Comment by Echolon** [comment]
Yes agree with Holger. I think its one of the most liked features here.

**Comment by iNPUTmice** [comment]
The way the app currently works it is impossible to implement this feature. Wanting this feature or agreeing that it is useful doesn’t change this.
The rewrite will probably just come with that from day 1. Based on the experience I have with Ltt.rs this will be much easier once I've switched frameworks.
So there is currently nothing to do.
